fossunited/api/schedule.py

```
@@ -97,7 +107,338 @@ def get_event_schedule(event_id: str) -> dict:
+        halls_list = local.request.args.getlist("halls")
+    else:
+        # fallback: parse comma-separated string params if provided
+        days_list = days.split(",") if days else []
+        halls_list = halls.split(",") if halls else []
+
+    # Parse days into date objects
+    days_date_objects = []
+    for d in days_list:
+        try:
+            days_date_objects.append(datetime.strptime(d.strip(), "%Y-%m-%d").date())
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)